redis_websocket_api/handler.py

```diff
@@ -15,18 +17,26 @@
 class WebsocketHandlerBase:
     """Define protocol for communication between web client and server."""
 
-    read_timeout = NotImplemented
     allowed_commands = NotImplemented
 
+    consumer_task: asyncio.Task  # for backwrds-compatibility, also in:
+    tasks: dict[str, asyncio.Task]  # Self-made TaskGroup for old Python
+
     def __init__(self, redis, websocket, read_timeout=None):
+        if read_timeout:
+            logger.warning(
+                "read_timeout is not used anymore because cleanup is trigered "
+                "immidiatly on connection loss"
```

Comment:
```suggestion
                "read_timeout is not used anymore because cleanup is triggered "
                "immediately on connection loss"
```